contracts/modules/oracle/Oracle.sol

```diff
@@ -31,10 +33,14 @@ contract Oracle is IOracle {
     }
 
     /**
-     * @param token - ERC20 token to get USD price for
+     * @param token_ - ERC20 token to get USD price for
+     * @dev - Weth is not supported by Chainlink Oracle, so treat it as Native Eth
      * @return price - the latest price in USD to 8 decimals
      */
-    function getLatestAnswer(address token) external returns (int256) {
+    function getLatestAnswer(address token_) external returns (int256) {
+        // TODO: test this
+        address token = token_ == LineLib.WETH ? Denominations.ETH : token_;
```

Comment:
This is going to cause issues testing on Goerli as the WETH address is different on Goerli vs Mainnet

Reply:
Can u add comment "We dont support native ETH but there is no WETH oracle. Use ETH price for WETH token"